src/controller.js

```diff
@@ -35,7 +35,7 @@ const newTableCellElement = () => {
 
 const createDisplayElement = num => {
     let img = document.createElement("img");
-    if (!num) return img;
```

Comment:
oh the number was coming as 0?

Reply:
yes and moreover we needed an image with name Alien0 so that it could take that img name